fRanz/tests/testthat/helper-kafka-vars.R

```diff
@@ -0,0 +1,2 @@
+KAFKA_HOST <- "localhost"
```

Comment:
don't you need a `docker-compose` call to ensure the kafka service is running at `localhost`?

Reply:
I'm checking these in now, and then will be adding a .travis.yml to do docker-compose and then test!

Reply:
I have verified they work locally